search/src/main/kotlin/com/ritense/search/service/SearchListColumnService.kt

```diff
@@ -61,4 +61,10 @@ class SearchListColumnService(
         with(findByOwnerIdAndKey(ownerId, key)) {
             this?.let { searchListColumnRepository.delete(it) }
         }
+
+    fun updateList(ownerId: String, searchListColumn: List<SearchListColumn>) {
+        var order = 0
```

Comment:
This can be replaced by `.mapIndexed` combined with the `.copy` method

Reply:
fixed